truenas_install/__main__.py

```diff
@@ -533,6 +534,9 @@ def main():
                     write_progress(0.58, "Configuring serial port")
                     configure_serial_port(root, os.path.join(root, "data/freenas-v1.db"))
 
+                if TRUENAS_VENDOR:
```

Comment:
I doubt this would work, this script is executed when the image is being installed and at that time it won't have access to this var. Have we tested this ?

Reply:
Also this should be done on Line 467 where other files in `{root}/data` are being created on fresh install. My suggestion is to just copy `/data` from the original system ane make this script agnostic of `.vendor` or other files we might want to copy from root in the future.